### PR TITLE
Incorporate Alfonso's recommended changes

### LINKED ISSUE #474: host-spec: incooperate Alfonso's feedback
https://w3f.github.io/polkadot-spec/develop/

This is something I should have done some time ago. I'll give some comments about the structure of this document to Florian.

### PR DESCRIPTION
**Pull request by lamafab**
Fixes #474.

Open considerations:

* [ ] Add examples to the trie structure?
* [ ] Move common definitions in the preliminaries (byte sequence, bitwise operators, etc.) to the appendix?
* [ ] Convert "Additional Information" chapter into an appendix?

### CODE REVIEW

File: 01_host/01_background/bootstrapping.adoc [hunk 1]
```diff
@@ -7,15 +7,15 @@ included to provide implementors with a pointer to what these steps are and
 where they are defined. In short, the following steps should be taken by all
 bootstrapping nodes:
 
-. The node needs to populate the state storage with the official genesis state,
+. The node must to populate the state storage with the official genesis state,
```
Comment (qdm12) [inline 01_host/01_background/bootstrapping.adoc]: Small typo 😉 

```suggestion
. The node must populate the state storage with the official genesis state,
```

File: 01_host/02_state/state_storage_trie.adoc [hunk 2]
```diff
@@ -389,10 +390,10 @@ stem:[H(N)] is defined as follows:
 [stem]
 ++++
 H: bbb B -> U_(i -> 0)^(32) bbb B_32\
-H(N): {(v_N,||v_N|| < 32 " and " N != R),("Blake2b"(v_n),||v_N|| >= 32 " or " N + R):}
+H(N): {(v_N,||v_N|| < 32 " and " N != R),("Blake2b"(v_n),||v_N|| >= 32 " or " N = R):}
```
Comment (qdm12) [inline 01_host/02_state/state_storage_trie.adoc]: I spotted the same in #508 ! 👍

File: 01_host/06_anv/collations.adoc [hunk 3]
```diff
@@ -6,7 +6,7 @@ chain validators. The Polkodat network protocol is agnostic on what candidate
 productionis mechanism each parachain uses and does not specify or mandate any
 of such production methods (e.g. BABE-GRANDPA, Aura, etc). Furthermore, the
 relay chain validator host implementation itself does not directly interpret or
-process teh internal transactions of the candidate, but rather rely on the
+process the internal transactions of the candidate, but rather rely on the
```
Comment (qdm12) [inline 01_host/06_anv/collations.adoc]: These misspellings should be fixed (and stay fixed) with https://github.com/w3f/polkadot-spec/pull/509 😉

**Review by FlorianFranzen — APPROVED** [review]
Looks good. Thank you!

>     * Add examples to the trie structure?

I would maybe start with a trie visualization. Feel free to open a new issue and hit Iggy up for it, if you want. Out of scope for this PR is would argue.
 
>     * Move common definitions in the preliminaries (byte sequence, bitwise operators, etc.) to the appendix?
>     *  Convert "Additional Information" chapter into an appendix?

I already started that for some of them, and I think we should continue to optimizes this and bring back the appendix, now that we no longer use multipage. Might be a good topic for a strategy meeting, i.e. general structure, now that we finally have one concise document.

**Pull request merged**